modules/unsupported/mbtiles/src/main/java/org/geotools/mbtiles/MBTilesMetadata.java

```diff
@@ -155,7 +191,11 @@ public void setBoundsStr(String boundsStr) {
             double miny = Double.parseDouble(matcherEnvelope.group(2));
             double maxx = Double.parseDouble(matcherEnvelope.group(3));
             double maxy = Double.parseDouble(matcherEnvelope.group(4));
-            setBounds(new ReferencedEnvelope(minx, maxx, miny, maxy, null));
```

Comment:
Now that this is surrounded by a try/catch, should we be throwing an exception up-stack to notify calling code that there was a failure?